examples/lessons/10/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "luma.gl-examples-shadowmap",
+  "description": "An es6 shadowmap example for luma.gl.",
+  "contributors": [
+    "Rye Terrell <ryeterrell@ryeterrell.net"
+  ],
+  "scripts": {
+    "start": "webpack-dev-server --progress --hot --open -d",
+    "start-local": "webpack-dev-server --env.local --progress --hot --open -d"
+  },
+  "dependencies": {
+    "luma.gl": ">=4.0.0-beta.6",
```

Comment:
luma.gl 4 is published now, we should use the official version.

Reply:
Done.